Revisar Protótipo Mobile - Tela_011

Gabrielx47 opened the pull request:
## Revisar o protótipo da tela_011:
- 1º passo: acessar o diretório com o arquivo da imagem (caminho: Modern-School-UI\Mobile\TelaDoProfessor\ManterDiárioDeClasse\ [Tela_011.png](https://github.com/Itatech-Jr/Modern-School-UI/blob/Branch_Gabriel33/Mobile/TelaDoProfessor/ManterDi%C3%A1rioDeClasse/Tela_011.png)). 
- 2º passo: verificar se a tela está conforme o design system do governo.

Gabrielx47 committed:
Create Tela_011.jpg

A tela_011 está em construção. Sendo assim, a mesma ainda não está finalizada. Done #33

Gabrielx47 committed:
Update Tela_011.jpg

Protótipo totalmente finalizado. Done #33

Gabrielx47 committed:
De JPEG para PNG

Correção do tipo de imagem utilizada. Estava usando JPEG e o correto é PNG. Done #33

joannestephany reviewed (changes requested):
A tela em si está ótima em relação a distribuição, mas as fontes(tamanho) estão muito pequenas, pense como pessoas com dificuldade de enxergar.

Gabrielx47 committed:
Mudanças realizadas. Done #33

Gabrielx47 committed:
Update Tela_011.png

Alguns componentes da Tela_011 não estavam presentes na imagem anterior, por consequência, o protótipo estava errado. Done #33

joannestephany reviewed (approved):
Aprovado, depois reviso a questão da quebra do texto, mas isso pode ser resolvido depois <3
